bpf_queue.c

```diff
@@ -164,6 +164,8 @@ ebpf_events_to_raw(struct ebpf_event_header *ev)
 		break;
 	case EBPF_EVENT_PROCESS_EXEC:
 		exec = (struct ebpf_process_exec_event *)ev;
+		if (exec->pids.tid != exec->pids.tgid)
```

Comment:
Why this change?

Reply:
So exec'ing, changes the tid to be the tgid?

Reply:
basically exec always comes from tgid, so tid is always equal tgid.
even if you do an exec from a child thread, at some point control passes to the main thread and there is where the exec happens.

Reply:
I think it's basically here:

https://elixir.bootlin.com/linux/v6.12.6/source/fs/exec.c#L1063-L1068

Reply:
Thanks, I was trying to find that.